### PR TITLE
feature/SKALE-2932 variable renaming

### PR DESCRIPTION
**Pull request by sergiy-skalelabs**
Configuration parameter and variable name renaming.

Shell variables:

CHAIN_NAME_MAINNET -> CHAIN_NAME_ETHEREUM
MAINNET_PROXY_PATH -> ETHEREUM_PROXY_PATH
INSECURE_PRIVATE_KEY_FOR_MAINNET -> INSECURE_PRIVATE_KEY_FOR_ETHEREUM
ACCOUNT_FOR_MAINNET -> ACCOUNT_FOR_ETHEREUM
NETWORK_FOR_MAINNET -> NETWORK_FOR_ETHEREUM
CID_MAINNET -> CID_ETHEREUM
URL_W3_MAIN_NET -> URL_W3_ETHEREUM

Config JSON variables:

insecureCommonBLSPublicKey -> commonBLSPublicKey
insecureBLSPublicKey -> BLSPublicKey

### CODE REVIEW

File: agent/README.md [hunk 1]
```diff
@@ -78,17 +78,17 @@ We will use networks called **MainNet** and **S-Chain** in this documentation:
 
 Fourth, export required environment variables:
 
-    export NETWORK_FOR_MAINNET="mainnet"
-    export INSECURE_PRIVATE_KEY_FOR_MAINNET="<YOUR_PRIVATE_KEY_HERE>"
+    export NETWORK_FOR_ETHEREUM="mainnet"
+    export INSECURE_PRIVATE_KEY_FOR_ETHEREUM="<YOUR_PRIVATE_KEY_HERE>"
```
Comment (cstrangedk) [inline agent/README.md]: this should be PRIVATE_KEY_FOR_ETHEREUM

File: agent/README.md [hunk 2]
```diff
@@ -78,17 +78,17 @@ We will use networks called **MainNet** and **S-Chain** in this documentation:
...
     export CHAIN_NAME_SCHAIN="Bob"
-    export URL_W3_MAIN_NET="http://127.0.0.1:8545"
+    export URL_W3_ETHEREUM="http://127.0.0.1:8545"
     export URL_W3_S_CHAIN="http://127.0.0.1:15000"
-    export ACCOUNT_FOR_MAINNET="<ACCOUNT_ADDRESS_HERE>"
+    export ACCOUNT_FOR_ETHEREUM="<ACCOUNT_ADDRESS_HERE>"
     export ACCOUNT_FOR_SCHAIN=""<ACCOUNT_ADDRESS_HERE>"
 
-Notice: ACCOUNT_FOR_MAINNET address corresponds to INSECURE_PRIVATE_KEY_FOR_MAINNET private key, ACCOUNT_FOR_SCHAIN address corresponds to INSECURE_PRIVATE_KEY_FOR_SCHAIN private key.
+Notice: ACCOUNT_FOR_ETHEREUM address corresponds to INSECURE_PRIVATE_KEY_FOR_ETHEREUM private key, ACCOUNT_FOR_SCHAIN address corresponds to INSECURE_PRIVATE_KEY_FOR_SCHAIN private key.
```
Comment (cstrangedk) [inline agent/README.md]: PRIVATE_KEY_FOR_ETHEREUM

File: agent/README.md [hunk 3]
```diff
@@ -125,30 +125,30 @@ You can check whether **IMA** is already bound with:
...
+        --url-main-net=$URL_W3_ETHEREUM \
         --url-s-chain=$URL_W3_S_CHAIN \
         --id-main-net=Mainnet \
         --id-s-chain=Bob \
         --cid-main-net=-4 \
         --cid-s-chain=-4 \
         --abi-main-net=../proxy/data/proxyMainnet.json \
         --abi-s-chain=../proxy/data/proxySchain_Bob.json \
-        --key-main-net=$INSECURE_PRIVATE_KEY_FOR_MAINNET \
+        --key-main-net=$INSECURE_PRIVATE_KEY_FOR_ETHEREUM \
```
Comment (cstrangedk) [inline agent/README.md]: PRIVATE_KEY_FOR_ETHEREUM

**Review by kladkogex — DISMISSED** [review]

**Review by payvint — APPROVED** [review]
Good!

**Review by kladkogex — APPROVED** [review]

**Review by cstrangedk — APPROVED** [review]

File: agent/README.md (hunk 3 above)
Comment (sergiy-skalelabs) [reply agent/README.md]: done

**Pull request merged**